content/docs/howto/configuration.md

```diff
@@ -198,8 +198,8 @@ The following adds a process with `type` equal to `hello` and makes it the defau
 echo "hello: echo hello world" > nodejs/yarn/Procfile
 pack build samples/nodejs \
   --path nodejs/yarn \
-  --buildpack gcr.io/paketo-buildpacks/nodejs \
-  --buildpack gcr.io/paketo-buildpacks/procfile \
+  --buildpack paketobuildpacks/nodejs:latest \
+  --buildpack gcr.io/paketo-buildpacks/procfile:latest \
```

Comment:
Does this `procfile` line need to be here still now that procfile is standard in the buildpack?

Reply:
I figured it's good to leave in since the docs here are explicitly about using a procfile for configuration. Though it might give users the wrong idea about what they have to do to get things working.